plugin-server/src/kafka/producer.ts

```diff
@@ -71,19 +79,20 @@ export const produce = async ({
     value,
     key,
     headers = [],
-    waitForAck = true,
+    waitForAck,
 }: {
     producer: RdKafkaProducer
     topic: string
     value: MessageValue
     key: MessageKey
     headers?: MessageHeader[]
-    waitForAck?: boolean
+    waitForAck: boolean
 }): Promise<number | null | undefined> => {
     status.debug('📤', 'Producing message', { topic: topic })
     const produceSpan = getSpan()?.startChild({ op: 'kafka_produce' })
     return await new Promise((resolve, reject) => {
         if (waitForAck) {
+            const ackTimer = ingestEventKafkaAckWait.labels(topic).startTimer()
```

Comment:
nit: we're also measuring the blocking `produce` call, that can block if the producer queue is full, but it's an OK approximation is long as we keep that in mind. To avoid someone misreading the metric, we could:
  - rename it `kafka_produce_latency`
  - tag it by `waitForAck` true/false
  - definitely keep it tagged by `topic`, we'll need that to filter out topics where we amortize the awaits by batching